.kokoro

```diff
@@ -187,6 +187,17 @@ upload_bazel_test_artifacts() {
   find -L . -name "sponge_log.xml" -type f | copy_to_artifacts
 }
 
+# Upgrades kokoro's current bazel version to at least 0.20.0
+upgrade_kokoro_bazel_if_needed() {
+  bazel version
+  # High sierra machines do not presently have a use_bazel.sh tool
+  # They do come pre-installed with bazel 0.21 however, so we don't need to update bazel.
+  if [ "$KOKORO_JOB_POOL" != "high-sierra" ]; then
```

Comment:
I'd prefer checking the result of `bazel version` parsing out the "build label". Is that possible? Then we don't depend on pool info (in 3 weeks the main pool switches to that environment).

Reply:
I've taken a new approach at this PR and removed the lines referred to here.